docs/user-manual/versions.adoc

```diff
@@ -31,6 +31,13 @@ These settings are now no longer required.
 +
 Configure `defaultMqttSessionExpiryInterval` instead.
 
+* Due to https://issues.apache.org/jira/browse/ARTEMIS-4532[ARTEMIS-4532] the names of addresses and queues related to MQTT topics and subscriptions respectively may change. This will only impact you:
++
+  . If the broker is configured to use a xref:wildcard-syntax.adoc[wildcard syntax] which doesn't match the xref:mqtt.adoc#wildcard-syntax[MQTT wildcard syntax].
+  . If you are using characters from the broker's wildcard syntax in your MQTT topic names or topic filters.
```

Comment:
It could be clearer from the text both items in the list need apply, just to drive it home

Reply:
Updated.